README.md

```diff
@@ -9,14 +9,14 @@ Build awesome list of the contributors of your project! You can even filter your
 In order to get the contributors list for your open-source project use:
 
 ```bash
-githubcontrib --owner USERNAME --repo REPO_NAME --cols 6 --filter user1,user2,user3 | pbcopy
+githubcontrib --owner USERNAME --repos REPO_NAME --cols 6 --filter user1,user2,user3 | pbcopy
 ```
 
 ### Advanced API
 
 | **Parameter name** | **Description**                                                   | **Default value**                      | **Sample value**                    |
 |--------------------|-------------------------------------------------------------------|----------------------------------------|-------------------------------------|
-| `repo`             | Repository name. ]                                                                      | (required)                                   | `--repo angular-material`              |
+| `repos`             | Repository name. ]                                                                      | (required)                                   | `--repo angular-material`              |
```

Comment:
Although the github endpoint is `repos`, I think, more semantically correct is to call the parameter `repo` since we're generating the list of contributors for only a single repository.